package.json

```diff
@@ -28,20 +28,18 @@
     "test-viewer": "yarn unit-viewer && jest lighthouse-viewer/test/viewer-test-pptr.js",
     "test-lantern": "bash lighthouse-core/scripts/test-lantern.sh",
     "unit-core": "jest \"lighthouse-core/\"",
-    "unit-core:ci": "jest --runInBand --coverage --ci \"lighthouse-core/\"",
-    "unit-cli": "jest  --runInBand \"lighthouse-cli/\"",
-    "unit-cli:ci": "jest --runInBand --coverage --ci \"lighthouse-cli/\"",
-    "unit-viewer": "jest \"lighthouse-viewer/\"",
+    "unit-cli": "jest \"lighthouse-cli/\"",
+    "unit-viewer": "jest lighthouse-viewer/**/*-test.js",
     "unit": "yarn unit-core && yarn unit-cli && yarn unit-viewer",
-    "unit:ci": "yarn unit-core:ci && yarn unit-cli:ci && yarn unit-viewer",
+    "unit:ci": "npm run unit-core -- --runInBand --ci --coverage && npm run unit-cli -- --runInBand --ci --coverage && npm run unit-viewer -- --runInBand --ci --coverage",
```

Comment:
this is a fun line to read :)

Reply:
> this is a fun line to read :)

ha, happy for ideas on how to make it more readable :)